src/applications/personalization/common/helpers.js

```diff
@@ -0,0 +1,58 @@
+export function makeUserObject(options = {}) {
```

Comment:
This was moved from the Dashboard e2e helpers file and support for a few additional options was added.